### PR TITLE
Added Twisted example and test in Travis

### PR DESCRIPTION
**Pull request by coryvirok**

### CODE REVIEW

File: THANKS.md [hunk 1]
```diff
@@ -13,3 +13,4 @@ Huge thanks to the following contributors (by github username). For the most up-
 - [tschieggm](https://github.com/tschieggm)
 - [zvirusz](https://github.com/zvirusz)
 - [isra17](https://github.com/isra17)
+- [juggernaut](https://github.com/juggernaut)
```
Comment (brianr) [inline THANKS.md]: This list should be in alphabetical order (the rest of it is, before the last two names)

**Comment by brianr** [comment]
:+1:

File: rollbar/examples/twisted/simpleserv.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
+
+# Copyright (c) Twisted Matrix Laboratories.
```
Comment (brianr) [inline rollbar/examples/twisted/simpleserv.py]: should this be here?

Comment (coryvirok) [inline rollbar/examples/twisted/simpleserv.py]: I believe so. I copied the code from Twisted so I left their copyright here.

**Pull request merged**